packages/old-accounter/src/data/save-transactions-to-db.ts

```diff
@@ -1,6 +1,7 @@
 import lodash from 'lodash';
 import moment from 'moment';
 import pg from 'pg';
+import type { CalTransaction } from '../../../modern-poalim-scraper/src/scrapers/types/cal/get-card-transactions-details.js';
```

Comment:
What are the benefits of using the old scraper code if you are not using any of the code itself?

I think it would be better to move that code somewhere else and run it separately.
In that new project you can use whatever you want, including Prisma

Reply:
Another example could be the crypto scrapers

Reply:
I use parts of the existing code for things like helper functions. And main reason to put it there is that's where the other scrapers were.

Reply:
Can we make this folder refactor another issue/PR?

Reply:
sure